config/locales/lt.yml

```
@@ -0,0 +1,49 @@
+lt:
+  language_name: Lithuanian
```

Would be good to have Lithuanian language name in Lithuanian
```suggestion
  language_name: Lietuvių 
```

---

Yeah, I thought about it but wasn't sure how you wanted it. Commited your suggestion, thanks!